webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/ArchiveApiTest.scala

```diff
@@ -110,8 +109,8 @@ class ArchiveApiTest extends Specification with AfterAll with Loggable {
   def children(f: File) = f.children.toList.map(_.name)
 
   // format: off
-//  org.slf4j.LoggerFactory.getLogger("application.archive").asInstanceOf[ch.qos.logback.classic.Logger].setLevel(ch.qos.logback.classic.Level.TRACE)
-//  org.slf4j.LoggerFactory.getLogger("configuration").asInstanceOf[ch.qos.logback.classic.Logger].setLevel(ch.qos.logback.classic.Level.TRACE)
+  org.slf4j.LoggerFactory.getLogger("application.archive").asInstanceOf[ch.qos.logback.classic.Logger].setLevel(ch.qos.logback.classic.Level.TRACE)
```

Comment:
The changes are ok !
But here maybe the trace logs should be kept as comments... :innocent: